devices/src/virtio/block.rs

```diff
@@ -707,13 +707,23 @@ mod tests {
 
             let epoll_config = EpollConfig::new(0, epoll_raw_fd, sender);
 
-            let f: File = tempfile().unwrap();
-            f.set_len(0x1000).unwrap();
+            let tmp_f = TempFile::new("/tmp/").unwrap();
```

Comment:
"`/tmp`" is very often used. We should probably define it in a constant or have a macro that calls `TempFile::new("/tmp/")`

Reply:
Created wrappers with the same name, but actually which defaults tmp files creation to "/tmp/". Please take a look: [utils/src/tempfile.rs](https://github.com/firecracker-microvm/firecracker/blob/ca185fa1c1a668a1f69cdb7012b40fd8a32a7216/utils/src/tempdir.rs) and [utils/src/tempdir.rs](https://github.com/firecracker-microvm/firecracker/blob/ca185fa1c1a668a1f69cdb7012b40fd8a32a7216/utils/src/tempdir.rs).